user_docs/en/userGuide.t2t

```diff
@@ -814,6 +814,26 @@ If you want to use these while still being able to use your cursor keys to read
 To toggle single letter navigation on and off for the current document, press NVDA+shift+space.
 %kc:endInclude
 
++++ Other navigation commands +++[OtherNavigationCommands]
+
+In addition to the quick navigation commands listed above, NVDA has commands that have no default keys assigned.
+To use these commands, you first need to assign gestures to them using the [Input Gestures dialog #InputGestures].
+Here is a list of available commands
+- Article
+- Grouping
+- Tab
+-
+Keep in mind that there are two commands for each type of element, for moving forward in the document and backward in the document, and you must assign gestures to both commands in order to be able to quickly navigate in both directions.
+
+For example, if you want to use the ``y`` / ``shift+y`` keys to quickly navigate through tabs, you would do the following
+
++ Open gestures dialog from browse mode.

```

Comment:
Should be "input gestures dialog", perhaps:
While in browse mode on a web site, open the Input Gestures dialog .

Reply:
```suggestion
+ Open input gestures dialog from browse mode.
```